dracut/30ignition/coreos-teardown-initramfs.sh

```diff
@@ -100,6 +100,23 @@ propagate_initramfs_hostname() {
     fi
 }
 
+# Persist automatic multipath configuration, if any.
+# When booting with `rd.multipath=default`, the default multipath
+# configuration is written. We need to ensure that the mutlipath configuration
+# is persisted to the final target.
+persist_automatic_multipath() {
+    if [ ! -f /sysroot/etc/multipath.conf ] && [ -f /etc/multipath.conf ]; then
+        echo "info: propagating automatic multipath configuration"
+        cp -v /etc/multipath.conf /sysroot/etc/
+        mkdir -p /sysroot/etc/multipath/multipath.conf.d
```

Comment:
This directly is created but empty?

Reply:
Yes. The directories need to exist, but can be empty.